core/src/main/java/hudson/PluginFirstClassLoader2.java

```diff
@@ -25,8 +25,9 @@ public class PluginFirstClassLoader2 extends URLClassLoader2 {
         registerAsParallelCapable();
     }
 
-    public PluginFirstClassLoader2(@NonNull URL[] urls, @NonNull ClassLoader parent) {
-        super(Objects.requireNonNull(urls), Objects.requireNonNull(parent));
+
+    public PluginFirstClassLoader2(String name, @NonNull URL[] urls, @NonNull ClassLoader parent) {
```

Comment:
class is annotated with `@Restricted(NoExternalUse.class)` which is not visible in the collapsed diff.